### PR TITLE
Fix routes and controllers for use with private labels namespace, and add flash notices

### PR DESCRIPTION
**Pull request by KBalderson**

### CODE REVIEW

File: app/helpers/private_labels/application_helper.rb [hunk 1]
```diff
@@ -7,5 +7,16 @@ def content_area_column_width_class
...
+    def alert_icon(level)
+      case level
+      when :notice
+        sanitize '<i class="icon icon-ok"></i>'
+      when :warning
+        sanitize '<strong>!</strong>'
+      when :error
+        sanitize '<i class="icon icon-remove"></i>'
+      end
+    end
```
Comment (dmkash) [inline app/helpers/private_labels/application_helper.rb]: Can we add a spec around this, please?

File: config/routes.rb [hunk 2]
```diff
@@ -14,9 +14,9 @@
 
   #Private Label Routes
   constraints PrivateLabelConstraint do
-    devise_for :people, :controllers => { :registrations => 'private_label/registrations', :confirmations => 'private_label/confirmations', :sessions => 'private_label/sessions', :omniauth_callbacks => "private_label/registrations/omniauth_callbacks", :passwords => 'passwords'},
+    devise_for :people, :controllers => { :registrations => 'private_labels/registrations', :confirmations => 'private_labels/confirmations', :sessions => 'private_labels/sessions', :omniauth_callbacks => "private_labels/registrations/omniauth_callbacks", :passwords => 'passwords'},
       :path_names => { :sign_in => 'login', :sign_out => 'logout', :password => 'secret', :confirmation => 'verification', :registration => 'register', :sign_up => 'new' }
-    namespace "private_label", path: '' do
+    namespace :private_labels, path: '' do
       root to: 'homepage#show'
```
Comment (dmkash) [inline config/routes.rb]: Nice catch.  Again, you fix the :poop: in my commits.

**Comment by dmkash** [comment]
@KBalderson This looks good to me!  Just need a spec around the new helper method.

**Comment by KBalderson** [comment]
gah! you're right!

**Comment by KBalderson** [comment]
Spec has been added

File: spec/helpers/private_labels/application_helper_spec.rb [hunk 3]
```diff
@@ -11,5 +11,19 @@ module PrivateLabels
...
+        expect(helper.alert_icon(:notice)).to include('icon-ok')
+      end
+      it 'returns the proper warning icon' do
+        expect(helper.alert_icon(:warning)).to include('!')
+      end
+      it 'returns the proper error icon' do
+        expect(helper.alert_icon(:error)).to include('icon-remove')
+      end
+    end
   end
```
Comment (dmkash) [inline spec/helpers/private_labels/application_helper_spec.rb]: :+1:

**Comment by dmkash** [comment]
Merging in!

**Pull request merged**